net/linknx/Makefile

```diff
@@ -19,7 +19,7 @@ PKG_MAINTAINER:=Othmar Truniger <github@truniger.ch>
 PKG_LICENSE:=GPL-2.0-or-later
 PKG_LICENSE_FILES:=LICENSE
 
-PKG_BUILD_DEPENDS:=USE_UCLIBC:argp-standalone USE_MUSL:argp-standalone
+PKG_BUILD_DEPENDS:=argp-standalone
```

Comment:
We also have glibc support right? So `USE_MUSL:` should be kept here.

Reply:
argp is a native library in glibc, argp-standalone is extracted from it.

Reply:
Maybe @rockdrilla can help with this one. :-)